src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -16,23 +16,29 @@ internal sealed class TupleErrorFieldSymbol : SynthesizedFieldSymbolBase
         private readonly TypeSymbol _type;
 
         /// <summary>
-        /// If this field represents a tuple element (including the name match), 
-        /// id is an index of the element (zero-based).
+        /// If this field represents a tuple element with index X
+        ///  2X      if this field represents Default-named element
+        ///  2X + 1  if this field represents Friendly-named element
         /// Otherwise, (-1 - [index in members array]);
```

Comment:
Are there cases where index is negative? If not, remove that part of the comment here and on `TupleElementIndex` property below.